mesh-delivery/README.md

```diff
@@ -42,22 +42,23 @@ If you don't have an account, [request a free trial on our website](https://www.
-- [Brightcove](web/brightcove/brightcove.html)
-- [Mpx](web/mpx/mpx.html)
-- [Akamai Adaptive Media Player](web/akamai-adaptive-media-player/akamai.html)
-- [Kaltura](web/kaltura/kaltura.html)
+- [Hls.js](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-hlsjs.html) ([démo](web/hls.js/hlsjs-wrapper.html))
+- [Dash.js](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-dashjs.html) ([démo](web/dash.js/dashjs-wrapper.html))
+- [ShakaPlayer](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-shaka-player.html) ([démo](web/shaka-player/shakaplayer-wrapper.html))
+- [RxPlayer](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-rx.html)
+- [VideoJS](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-videojs.html) ([démo](web/video.js/hlsjs-videojs.html))
+- [Bitmovin](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-bitmovin.html) ([démo](web/bitmovin/bitmovin.html))
+- [THEOplayer](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-theoplayer.html) ([démo](web/theoplayer/theoplayer.html))
+- [JWPlayer](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-jwplayer.html) ([démo](web/jwplayer/hlsjs-jwplayer.html))
+- [Azure Media Player](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-azure-media-player.html) ([démo](web/azure-media-player/azuremp.html))
+- [Castlabs](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-castlabs.html) ([démo](web/castlabs/castlabs.html))
+- [Clappr](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-clappr.html) ([démo](web/clappr/hlsjs-clappr.html))
+- [FlowPlayer](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-flowplayer.html) ([démo](web/flowplayer/flowplayer.html))
+- [Radiant Media Player](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-radiant.html) ([démo](web/radiant-media-player/radiant.html))
+- [Brightcove](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-brightcove.html) ([démo](web/brightcove/brightcove.html))
+- [Mpx](https://www.lumen.com/help/en-us/cdn/mesh-delivery-for-streaming/mesh-delivery-web/integration-for-web/mesh-delivery-mpx.html) ([démo](web/mpx/mpx.html))
```

Comment:
Is MPX still the name? I remember some of these players changed names. Check my feedback on Sanket's doc migration tracker Conflu doc.